### ISSUE TITLE
Containers get created in a stopped state.

### ISSUE THREAD

**Issue by renier** [chef/chef-provisioning-docker#24]
Containers get created in a stopped state. Starting them just returns them to a stopped state.

```ruby
require 'chef/provisioning/docker_driver'

with_chef_server "http://some.address.com:8889"

machine 'mario' do
  recipe 'nginx'

  machine_options :docker_options => {
    :base_image => {
      :name => 'ubuntu',
      :repository => 'ubuntu',
      :tag => '14.04'
    },
    :detached => true
  }
end
```

I have tried adding a `:command` like `tail -f /etc/lsb-release`, but to no avail.

**Comment by marc-** [comment]
This happens due to \:port option is missing. It is eventually bug.

**Comment by randomcamel** [comment]
Should be fixed by PR #47.
